test/test-flushable.rb

```diff
@@ -46,4 +46,12 @@ def test_flush_not_recursive
       table.flush(:recursive => false)
     end
   end
+
+  def test_flush_dependent
+    table = Groonga::Hash.create
+    table.extend(Groonga::Flushable)
+    assert_nothing_raised do
+      table.flush(:dependent => true)
```

Comment:
Does this really call `grn_obj_flush_dependent()`?

Reply:
Sorry.　I don't know what you mean.
Could you explain it more detail?

Does that mean that it's wrong function name as below?
`grn_obj_flush_dependent()` -> `grn_obj_flush_recursive_dependent`

Reply:
`rb_recursive_p` is `true` by default: 
https://github.com/ranguba/rroonga/pull/160/files#diff-51d9f7b0ed10a8277d8a28a31b3c6b2eR55
If `rb_recursive_p` is `true`, we don't use `rb_dependent_p`: https://github.com/ranguba/rroonga/pull/160/files#diff-51d9f7b0ed10a8277d8a28a31b3c6b2eR69
This line doesn't specify `:recursive => `false`.

Anyway, this test isn't good. Because it's passed without confirming what is occurred.